assets/js/components/UserMenu.js

```diff
@@ -182,7 +184,7 @@ export default function UserMenu() {
 					aria-expanded={ menuOpen }
 					aria-controls="user-menu"
 				>
-					{ userEmail }
+					{ unifiedDashboardEnabled ? undefined : userEmail }
```

Comment:
I think if we do it like this, it will be easier to read:

```suggestion
					{ ! unifiedDashboardEnabled && userEmail }
```